acceptance/suites/tests/555-puppetserver-service/reboot.rb

```diff
@@ -0,0 +1,63 @@
+if (options[:type] == 'pe') then
+  pssn='pe-puppetserver'
+else
+  pssn='puppetserver'
+end
+
+test_name "Validates the ability of the #{pssn} service to reliably start after a reboot."
+
+max=10
+sleeptime=40
+retryCounter=0
+step "\n-----------\nValidate #{pssn} is up and running, then reboot, repeat #{max} times\n-----------"
+
+for i in 1..max 
```

Comment:
The primary purpose of the loop is to reboot the system under test <max> times, and to validate that the puppetserver service is up and running after the reboot.